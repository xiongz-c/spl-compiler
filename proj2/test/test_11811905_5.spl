struct test1 {};

struct test2 {
    int field1[2];
    struct test1 field2;
} v1, v2[2];

int test_11811806_5() {
    float v3 = 1.0;
    int v4[3][2];
    struct test3 v5;
    struct test2 local;
    v1.field2 = local;
    v1.field1[0] = v3;
    v1.field1 = v4[2];
    v2[0].field3 = v6;

    return 0;
}