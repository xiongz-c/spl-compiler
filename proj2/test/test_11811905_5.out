Error type 14 at Line 11: no such member
Error type 5 at Line 14: unmatching type on both sides of assignment
Error type 14 at Line 16: no such member: field3
Error type 1 at Line 16: undefined variable: v6
